contributors/devel/sig-scalability/kubemark-setup-guide.md

```diff
@@ -0,0 +1,74 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster given that a base cluster (to run hollow-node pods) and separate master instance(s) (to act as master(s) for the hollow nodes) are already present.
+
+## Precondition
+You need kubemark master(s) and external cluster to set up a kubemark cluster.
+
+The functions are as follows:
+
+- kubemark master(s): it's used to be the kubemark cluster's masters
```

Comment:
nit: Instead of saying `master(s)` everywhere in the doc, can you simply just say master and mention a note at the top that it can even be a set of masters (i.e HA setup)?

Reply:
Done

Reply:
You still seem to be using `master(s)` everywhere in the doc. Given that you already mentioned it can be single or HA, can you simply replace those with just `master`?

Reply:
Done